### PR TITLE
Update experimental docs dsb -> dab

### PR DESCRIPTION
**Pull request by vdemeester**
Follow-up to #23945, fixing bundle extension in experimental documentation 🐄.

/cc @thaJeztah @dnephin @aanand 

🐸

Signed-off-by: Vincent Demeester vincent@sbr.pm

### CODE REVIEW

**Comment by dnephin** [comment]
LGTM

File: experimental/docker-stacks-and-bundles.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ Run 'docker stack COMMAND --help' for more information on a command.
 
 Distributed application bundles are described in a JSON format. When bundles
 are persisted as files, the file extension is `.dab` (Docker 1.12RC2 tools use
-`.dsb` for the file extension—this will be updated in the next release client).
+`.dab` for the file extension—this will be updated in the next release client).
```
Comment (thaJeztah) [inline experimental/docker-stacks-and-bundles.md]: This one should probably remain `dsb`

**Comment by thaJeztah** [comment]
one comment, but LGTM once addressed

File: experimental/docker-stacks-and-bundles.md (hunk 1 above)
Comment (vdemeester) [inline experimental/docker-stacks-and-bundles.md]: oh :sweat_smile:

**Comment by thaJeztah** [comment]
LGTM, thanks!

**Pull request merged**